openmcyclus/DepleteReactor.py

```diff
@@ -0,0 +1,11 @@
+from cyclus.agents import Facility 
+
+class DepleteReactor(Facility):
+    '''
+    Sample facility to get things going
+    '''
```

Comment:
Let's also rewrite this docstring. Maybe something like "This class represents a reactor, and includes methods that allow depletion via coupling to OpenMC's deplete module"

Reply:
I was hoping to do that in another PR, when I add more functionality to the archetype. This PR was mostly to get some files into the repo. If you think it's important for this to be added to this PR, I can do that.

Reply:
I think you should do it here.